libs/application/template-api-modules/src/lib/modules/templates/health-insurance/health-insurance.utils.ts

```diff
@@ -119,6 +119,11 @@ export const transformApplicationToHealthInsuranceDTO = (
     previousIssuingInstitution:
       extractAnswer(application.answers, 'formerInsurance.institution') ?? '',
     isHealthInsuredInPreviousCountry:
+      extractAnswer(application.answers, 'formerInsurance.registration') ==
+      'yes'
+        ? 1
+        : 0,
+    hasHealthInsuranceRightInPreviousCountry:
       extractAnswer(application.answers, 'formerInsurance.entitlement') == 'yes'
```

Comment:
Same as above. Will update the field in frontend and make sure it changes here as well.